### PR TITLE
Export structurizr

### PR DESCRIPTION
**Pull request by mh182**
This is a draft PR to discuss the implementation for #1016.

### CODE REVIEW

File: scripts/exportStructurizr.gradle [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// * refactor: extract configuration paramaters
+// * provide configuration parameters
+
+//tag::exportStructurizr[]
+task exportStructurizr (
+        group: 'docToolchain',
+        description: 'exports the views of a Structurizr DSL file to diagramms'
+) {
+
+    def workspaceFile = findProperty("structurizr.workspace")?:config.structurizr.workspace
```
Comment (rdmueller) [inline scripts/exportStructurizr.gradle:38]: that's great that it will also work with config properties!

**Review by rdmueller — APPROVED** [review]
looks good!

**Comment by mh182** [comment]
I think I have finished the implementation part of this ticket. 
Now I have the following questions:
* Cooperation workflow: I rebased my local git branch (not yet pushed) to the `ng` branch. This means I would force push the changes to my fork [`mh182:exportStructurizr`](https://github.com/mh182/docToolchain/tree/exportStructurizr), which this review is based on.
    * How do we make a new review-cycle?
    * Is this the right way how to do review cycles on the same PR?
* I started writing the task documentation. How can I generate the documentation shown on http://doctoolchain.org locally on my machine? The developer documentation provides how it is done via the GitHub interface, which is not the right way to do it when I develop.

**Comment by mh182** [comment]
I screwed up this PR. My branch is rebased on ng and includes also the ADR.
I think I will just close this PR and create a new one agains the ng branch. 
My question is how do we handle the ADR (which IMHO is not ready yet)?

**Comment by rdmueller** [comment]
* rebase can also be done through the github ui
* if the ADR is part of the PR, not problem. It is marked as draft
* generation of docs should be just `./dtcw generateSite`

**Comment by rdmueller** [comment]
and regarding the docs - the last netlify task in a PR is a link to preview-docs

**Pull request closed without merging**